### ISSUE TITLE
cokit compile failed on Xcode 11

### ISSUE THREAD

**Issue by Magic-Unique** [alibaba/coobjc#93]
### Context and Description

Use CocoaPods install cokit. Compile on Xcode 11.

UIImagePickerController+Coroutine.h:

```objc
// line 142
@property(nonatomic, strong, readonly) PHLivePhoto *livePhoto PHOTOS_AVAILABLE_IOS_TVOS(9_1, 10_0);

                                                             ^

                                                             Expected ';' at end of declaration list
```

### Environment Details

* coobjc version:
cocore (1.2.5)
cokit (1.2.1)
coobjc (1.2.0)
* iOS version: iOS 13
* Xcode version: Xcode 11
* Other informations: None

### Expected behavior

Compile Success.

### Actual behavior

Compile Failed.

### Steps to Reproduce

Compile with Xcode 11.

**Comment by lgb1234a** [comment]
+1

**Comment by SuperMarioBean** [comment]
same here

**Comment by SuperMarioBean** [comment]
似乎是因为 Apple 移除了原来的 PHOTOS_AVAILABLE_IOS_TVOS 宏定义, 麻烦各位修改下

这是之前 xcode 10 的定义
```
typedef NS_ENUM(NSInteger, PHAssetCollectionSubtype) {
    
    // PHAssetCollectionTypeAlbum regular subtypes
    PHAssetCollectionSubtypeAlbumRegular         = 2,
    PHAssetCollectionSubtypeAlbumSyncedEvent     = 3,
    PHAssetCollectionSubtypeAlbumSyncedFaces     = 4,
    PHAssetCollectionSubtypeAlbumSyncedAlbum     = 5,
    PHAssetCollectionSubtypeAlbumImported        = 6,
    
    // PHAssetCollectionTypeAlbum shared subtypes
    PHAssetCollectionSubtypeAlbumMyPhotoStream   = 100,
    PHAssetCollectionSubtypeAlbumCloudShared     = 101,
    
    // PHAssetCollectionTypeSmartAlbum subtypes
    PHAssetCollectionSubtypeSmartAlbumGeneric    = 200,
    PHAssetCollectionSubtypeSmartAlbumPanoramas  = 201,
    PHAssetCollectionSubtypeSmartAlbumVideos     = 202,
    PHAssetCollectionSubtypeSmartAlbumFavorites  = 203,
    PHAssetCollectionSubtypeSmartAlbumTimelapses = 204,
    PHAssetCollectionSubtypeSmartAlbumAllHidden  = 205,
    PHAssetCollectionSubtypeSmartAlbumRecentlyAdded = 206,
    PHAssetCollectionSubtypeSmartAlbumBursts     = 207,
    PHAssetCollectionSubtypeSmartAlbumSlomoVideos = 208,
    PHAssetCollectionSubtypeSmartAlbumUserLibrary = 209,
    PHAssetCollectionSubtypeSmartAlbumSelfPortraits PHOTOS_AVAILABLE_IOS_TVOS(9_0, 10_0) = 210,
    PHAssetCollectionSubtypeSmartAlbumScreenshots PHOTOS_AVAILABLE_IOS_TVOS(9_0, 10_0) = 211,
    PHAssetCollectionSubtypeSmartAlbumDepthEffect PHOTOS_AVAILABLE_IOS_TVOS(10_2, 10_1) = 212,
    PHAssetCollectionSubtypeSmartAlbumLivePhotos PHOTOS_AVAILABLE_IOS_TVOS(10_3, 10_2) = 213,
    PHAssetCollectionSubtypeSmartAlbumAnimated PHOTOS_AVAILABLE_IOS_TVOS(11_0, 11_0) = 214,
    PHAssetCollectionSubtypeSmartAlbumLongExposures PHOTOS_AVAILABLE_IOS_TVOS(11_0, 11_0) = 215,
    // Used for fetching, if you don't care about the exact subtype
    PHAssetCollectionSubtypeAny = NSIntegerMax
} PHOTOS_ENUM_AVAILABLE_IOS_TVOS(8_0, 10_0);
```

这是 xcode 11.1 的定义
```
typedef NS_ENUM(NSInteger, PHAssetCollectionSubtype) {
    
    // PHAssetCollectionTypeAlbum regular subtypes
    PHAssetCollectionSubtypeAlbumRegular         = 2,
    PHAssetCollectionSubtypeAlbumSyncedEvent     = 3,
    PHAssetCollectionSubtypeAlbumSyncedFaces     = 4,
    PHAssetCollectionSubtypeAlbumSyncedAlbum     = 5,
    PHAssetCollectionSubtypeAlbumImported        = 6,
    
    // PHAssetCollectionTypeAlbum shared subtypes
    PHAssetCollectionSubtypeAlbumMyPhotoStream   = 100,
    PHAssetCollectionSubtypeAlbumCloudShared     = 101,
    
    // PHAssetCollectionTypeSmartAlbum subtypes
    PHAssetCollectionSubtypeSmartAlbumGeneric    = 200,
    PHAssetCollectionSubtypeSmartAlbumPanoramas  = 201,
    PHAssetCollectionSubtypeSmartAlbumVideos     = 202,
    PHAssetCollectionSubtypeSmartAlbumFavorites  = 203,
    PHAssetCollectionSubtypeSmartAlbumTimelapses = 204,
    PHAssetCollectionSubtypeSmartAlbumAllHidden  = 205,
    PHAssetCollectionSubtypeSmartAlbumRecentlyAdded = 206,
    PHAssetCollectionSubtypeSmartAlbumBursts     = 207,
    PHAssetCollectionSubtypeSmartAlbumSlomoVideos = 208,
    PHAssetCollectionSubtypeSmartAlbumUserLibrary = 209,
    PHAssetCollectionSubtypeSmartAlbumSelfPortraits API_AVAILABLE(ios(9)) = 210,
    PHAssetCollectionSubtypeSmartAlbumScreenshots API_AVAILABLE(ios(9)) = 211,
    PHAssetCollectionSubtypeSmartAlbumDepthEffect API_AVAILABLE(macos(10.13), ios(10.2), tvos(10.1)) = 212,
    PHAssetCollectionSubtypeSmartAlbumLivePhotos API_AVAILABLE(macos(10.13), ios(10.3), tvos(10.2)) = 213,
    PHAssetCollectionSubtypeSmartAlbumAnimated API_AVAILABLE(macos(10.15), ios(11), tvos(11)) = 214,
    PHAssetCollectionSubtypeSmartAlbumLongExposures API_AVAILABLE(macos(10.15), ios(11), tvos(11)) = 215,
    PHAssetCollectionSubtypeSmartAlbumUnableToUpload API_AVAILABLE(macos(10.15), ios(13), tvos(13)) = 216,

    // Used for fetching, if you don't care about the exact subtype
    PHAssetCollectionSubtypeAny = NSIntegerMax
};
```

**Comment by sevenOnClass** [comment]
> ### Context and Description
> Use CocoaPods install cokit. Compile on Xcode 11.
> 
> UIImagePickerController+Coroutine.h:
> 
> ```objc
> // line 142
> @property(nonatomic, strong, readonly) PHLivePhoto *livePhoto PHOTOS_AVAILABLE_IOS_TVOS(9_1, 10_0);
> 
>                                                              ^
> 
>                                                              Expected ';' at end of declaration list
> ```
> 
> ### Environment Details
> * coobjc version:
>   cocore (1.2.5)
>   cokit (1.2.1)
>   coobjc (1.2.0)
> * iOS version: iOS 13
> * Xcode version: Xcode 11
> * Other informations: None
> 
> ### Expected behavior
> Compile Success.
> 
> ### Actual behavior
> Compile Failed.
> 
> ### Steps to Reproduce
> Compile with Xcode 11.
> 
> ### method call stacks

Adapt Xcode11: replace PHOTOS_AVAILABLE_IOS_TVOS(9_1, 10_0) with  API_AVAILABLE(ios(9.1), tvos(10))

**Comment by sevenOnClass** [comment]
> Adapt Xcode11: 

replace `PHOTOS_AVAILABLE_IOS_TVOS(9_1, 10_0)` with `API_AVAILABLE(ios(9.1), tvos(10))`

**Comment by sevenOnClass** [comment]
use the following code  in podfile
```
  pod 'cocore', :git => 'https://github.com/alibaba/coobjc.git',:commit =>'83f28d0b'#,'~> 1.2.5'
  pod 'coobjc', :git => 'https://github.com/alibaba/coobjc.git',:commit =>'83f28d0b'#,'~> 1.2.0'
  pod 'cokit', :git => 'https://github.com/alibaba/coobjc.git',:commit =>'83f28d0b'#,'~> 1.2.1'
```

**Comment by NianJi** [comment]
fix in cokit 1.2.6,
